src/Serilog/Parameters/PropertyValueConverter.cs

```diff
@@ -186,6 +201,23 @@ LogEventPropertyValue CreatePropertyValue(object value, Destructuring destructur
             return new ScalarValue(value.ToString());
         }
 
+        private LogEventPropertyValue Stringify(object value)
```

Comment:
Nitpick (can pick this up later): the codebase doesn't use any explicit `private`/`internal` modifiers.